integration-sqs-to-lambda/main.rs

```diff
@@ -0,0 +1,27 @@
+use aws_lambda_events::event::sqs::SqsEvent;
+use lambda_runtime::{run, service_fn, Error, LambdaEvent};
+
+static UNDEFINED: &str = "undefined";
+
+async fn function_handler(event: LambdaEvent<SqsEvent>) -> Result<(), Error> {
+
+    event.payload.records.iter().for_each(|record| {
+        // process the record
+        tracing::info!("Message body: {}", record.body.as_deref().unwrap_or(UNDEFINED))
```

Comment:
It is good enough to use default value for the body. If people want to handle `None`, they can do it with match on body. 

```rust
        let body = record.body.unwrap_or_default();
        // process the record
        tracing::info!("Message body: {}", body)
```

Reply:
the `body` is an `Option<String>` and does not implement `std:fmt:Display`. I can do something like
```rust
tracing::info!("Message body: {}", record.body.as_deref().unwrap_or_default())
```
Let me know if this works